src/database/queries/tag.js

```diff
@@ -0,0 +1,21 @@
+import knex from '../knex'
+import * as _ from './utilities'
+
+const add = attributes =>
+  _.createRecord( 'tag', attributes )
+
+const deleteAll = () =>
+  _.deleteAll( 'tag' )
+
+const getAll = () =>
+  _.findAll( 'tag' )
+  .orderBy('names', 'asc')
+
+const getBy = ( column, data ) =>
+  _.findAllWhere( 'tag', column, data )
```

Comment:
Shouldn't this have a `.first()`? I think you want this to yield a single record or null as apposed to an array of 1 record or an empty array.